### ISSUE TITLE
Unknown function netrw#LocalBrowseCheck in NVIM v0.6.0-dev

### ISSUE THREAD

**Issue by IYaksha** [kyazdani42/nvim-tree.lua#630]
Thanks for the work you have put in ```NvimTree```.
I am witnessing the following error when opening neovim and when toggling ```NvimTree```:

```
E5108: Error executing lua ...e/pack/packer/start/nvim-tree.lua/lua/nvim-tree/view.lua:338: Vim(call):E117: Unknown
function: netrw#LocalBrowseCheck
```

Neovim version - ```NVIM v0.6.0-dev```

**Comment by kyazdani42** [comment]
Hi!
could you provide a minimal configuration to reproduce this please ?

**Comment by Iron-E** [comment]
Related to #605?

**Comment by IYaksha** [comment]
@kyazdani42 Sorry for my super late reply. I created a very small lua config in order to reproduce the issue I was facing. I also updated my nightly neovim and now there are no errors. Also, issue was on my part because in README.md, the installation setup via packer has changed which I didn't read. So sorry for raising this issue. I am closing this issue. Thanks for taking time to reply on the issue.
